app/Console/Commands/CreatePostingPeriodCommand.php

```
@@ -0,0 +1,27 @@
+use App\Services\PostingPeriodService;
+use App\Models\PostingPeriod;
+
+class CreatePostingPeriodCommand extends Command
+{
+    protected $signature = 'posting-period:create';
+    protected $description = 'Create posting period for the next month';
+
+    public function handle(PostingPeriodService $service): int
+    {
+        try {
+            $result = $service->createPostingPeriod();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (Unused Code Rules)

(UnusedLocalVariable)